ui/packages/catalog-ui-search/src/main/webapp/react-component/styles/mixins/is-button.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import { css } from '../styled-components'
+
+export const IsButton = theme => css`
```

Comment:
Should we be using the [react button component](https://github.com/codice/ddf/blob/d012b0946f68f82166896ebf66fdf09cd00dd60c/ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/button/button.tsx) instead of this?

Reply:
I was wondering that as well

Reply:
Update: the color and highlighting behavior I need doesn't work well with this component